aidants_connect_web/templatetags/ac_extras.py

```diff
@@ -1,12 +1,25 @@
+import json
 from re import sub as re_sub
 
 from django import template
 from django.template import Library
-from django.template.base import Node, Parser, NodeList, TextNode, Token
+from django.template.base import Node, NodeList, Parser, TextNode, Token
+
+from aidants_connect_web.models import Usager
 
 register = template.Library()
 
 
+@register.filter
+def json_search_value(user: Usager):
+    return json.dumps(
+        [
+            user.preferred_username if user.preferred_username else user.family_name,
```

Comment:
```suggestion
            user.preferred_username or user.family_name,
```

Reply:
En fait je pense qu'il faudrait rechercher aussi sur `user.family_name` même si le `user.preferred_username` est défini. Donc que Madame Truc née Machin remonte dans la recherche sur `Mach` et aussi sur `Truc`.

Mais si on fait comme ça : 

```suggestion
            user.preferred_username,
            user.family_name,
```

Dans le cas où le `preferred_username` est vide, est-ce que la recherche va mal se comporter ?